src/Language/Pirouette/Example/QuasiQuoter.hs

```diff
@@ -50,6 +58,11 @@ trQ f = case runExcept f of
   Left err -> fail err
   Right r -> return r
 
+maybeQ :: (Show e) => Either e a -> Q a
```

Comment:
I think this could benefit from pretty-printing the type error, but that should perhaps be an issue and a separate PR later on :thinking:

Reply:
Since only us two are working on this, I suggest to make it part of this PR. I'll take care of that tomorrow.